test(e2e,playwright): show.tsx

anand-harsh opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->
 #51705
<!-- Feel free to add any additional description of changes below this line -->

![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/94885893/a9b57c9a-b80d-46e6-a7c0-618acf35be09)

I was getting above warning so I splitted the code into two seperate files.

naomi-lgbt commented on e2e/show-button.spec.ts:
```diff
@@ -0,0 +1,37 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
```
```suggestion
```

Don't need to declare this, because...

anand-harsh commented:
Hi @naomi-lgbt , I have tried to resolve all issues. Kindly review and suggest if something more needed.
Without seed Title test is getting failed in chromium. So I added the seed only ion title and removed from other one. Thank you for helping with the issue.

anand-harsh commented:
Hi @naomi-lgbt , I have tried to resolve all issues. Kindly review and suggest if something more needed.
Without seed,  Title test is getting failed in chromium. So I added the seed only in title and removed from other one. Thank you for helping with the issue.

anand-harsh commented:
Hi @naomi-lgbt , I have resolved the issues. Kindly review and suggest if something more needed.
Without seed, Title test is getting failed in chromium. So I added the seed only in title and removed from other one. Thank you for helping with the issue.

jdwilkin4 commented:
HI @anand-harsh ! 

I removed the closes keyword from your PR because we don't want to close the hacktoberfest issue because it needs to stay open for multiple contributors 

You should only use the closes keyword when you want to automatically close an issue after the PR has been approved and merged.
In this case, we only want to simply mention it.

Thanks for understanding 👍

anand-harsh commented:
> HI @anand-harsh !
> 
> I removed the closes keyword from your PR because we don't want to close the hacktoberfest issue because it needs to stay open for multiple contributors
> 
> You should only use the closes keyword when you want to automatically close an issue after the PR has been approved and merged. In this case, we only want to simply mention it.
> 
> Thanks for understanding 👍

Sure Sir

anand-harsh commented:
Hi @naomi-lgbt, @huyenltnguyen , I have resolved the issues. Kindly review and suggest if something more needed.
Without seed, Title test is getting failed in chromium. So I added the seed only in title and removed from other one. Thank you for helping with the issue.

huyenltnguyen reviewed (commented):
Looking at this again, I think we should consolidate the test files instead of having one for the button and one for the title.

anand-harsh commented:
> Looking at this again, I think we should consolidate the test files instead of having one for the button and one for the title.

Done this change. I consolidated both files in a single file and it is running now.

anand-harsh commented:
> Looking at this again, I think we should consolidate the test files instead of having one for the button and one for the title.

Hi @huyenltnguyen , Done this change. I consolidated both files in a single file and it is running now. Kindly check and suggest if more changes needed.

anand-harsh commented:
@huyenltnguyen I did the changes and deleted seed and beforAll.

naomi-lgbt reviewed (approved):
A quick note that you don't need to continuously merge `main` into this branch.

Also, you shouldn't work directly off of your `main` branch.

anand-harsh commented:
> A quick note that you don't need to continuously merge `main` into this branch.
> 
> Also, you shouldn't work directly off of your `main` branch.

Sure, I will take care of this.
